src/test/java/io/confluent/examples/streams/kafka/EmbeddedSingleNodeKafkaCluster.java

```diff
@@ -84,42 +82,39 @@ public EmbeddedSingleNodeKafkaCluster(final Properties brokerConfig) {
    */
   public void start() throws Exception {
     log.debug("Initiating embedded Kafka cluster startup");
-    log.debug("Starting a ZooKeeper instance...");
-    zookeeper = new ZooKeeperEmbedded();
-    log.debug("ZooKeeper instance is running at {}", zookeeper.connectString());
 
-    final Properties effectiveBrokerConfig = effectiveBrokerConfigFrom(brokerConfig, zookeeper);
-    log.debug("Starting a Kafka instance on port {} ...",
-      effectiveBrokerConfig.getProperty(SocketServerConfigs.LISTENERS_CONFIG));
+    final Map<String, String> effectiveBrokerConfig = effectiveBrokerConfigFrom(brokerConfig);
+    log.debug(
+        "Starting a Kafka instance on port {} ...",
+        effectiveBrokerConfig.get(SocketServerConfigs.LISTENERS_CONFIG)
+    );
     broker = new KafkaEmbedded(effectiveBrokerConfig);
-    log.debug("Kafka instance is running at {}, connected to ZooKeeper at {}",
-      broker.brokerList(), broker.zookeeperConnect());
+    log.debug("Kafka instance is running at {}", broker.brokerList());
+
+    final Properties adminConfig = new Properties();
+    adminConfig.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, broker.brokerList());
+    admin = Admin.create(adminConfig);
 
     final Properties schemaRegistryProps = new Properties();
 
     schemaRegistryProps.put(SchemaRegistryConfig.KAFKASTORE_TIMEOUT_CONFIG, KAFKASTORE_OPERATION_TIMEOUT_MS);
     schemaRegistryProps.put(SchemaRegistryConfig.DEBUG_CONFIG, KAFKASTORE_DEBUG);
     schemaRegistryProps.put(SchemaRegistryConfig.KAFKASTORE_INIT_TIMEOUT_CONFIG, KAFKASTORE_INIT_TIMEOUT);
-    schemaRegistryProps.put(SchemaRegistryConfig.KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG, effectiveBrokerConfig.getProperty(SocketServerConfigs.LISTENERS_CONFIG));
+    schemaRegistryProps.put(SchemaRegistryConfig.KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG, broker.brokerList());
 
     schemaRegistry = new RestApp(0, null, KAFKA_SCHEMAS_TOPIC, AVRO_COMPATIBILITY_TYPE, schemaRegistryProps);
     schemaRegistry.start();
     running = true;
   }
 
-  private Properties effectiveBrokerConfigFrom(final Properties brokerConfig, final ZooKeeperEmbedded zookeeper) {
-    final Properties effectiveConfig = new Properties();
-    effectiveConfig.putAll(brokerConfig);
-    effectiveConfig.put(ZkConfigs.ZK_CONNECT_CONFIG, zookeeper.connectString());
-    effectiveConfig.put(ZkConfigs.ZK_SESSION_TIMEOUT_MS_CONFIG, 30 * 1000);
-    effectiveConfig.put(SocketServerConfigs.LISTENERS_CONFIG, String.format("PLAINTEXT://127.0.0.1:%s", DEFAULT_BROKER_PORT));
```

Comment:
Needed to remove this -- otherwise, I got some config errors about other missing config... not sure why this was set to begin with? Did not want to go down the rabbit whole figuring out all other required/dependent configs to keep it (also not sure why it did work with the old setup, but not the new one... 🤷)